macros/LearnSidebar.ejs

```diff
@@ -2032,6 +2095,21 @@ var text = mdn.localStringMap({
         </ol>
     </details>
   </li>
+  <li class="toggle">
+    <details <%=currentPageIsUnder('Tools_and_testing/Client-side_JavaScript_frameworks')%>>
+        <summary><%=text['Svelte']%></summary>
+        <ol>
+          <li><a href="<%=baseURL%>Tools_and_testing/Client-side_JavaScript_frameworks/Svelte_getting_started"><%=text['Getting_started_with_Svelte']%></a></li>
+          <li><a href="<%=baseURL%>Tools_and_testing/Client-side_JavaScript_frameworks/Svelte_Todo_list_beginning"><%=text['Starting_our_Svelte_Todo_list_app']%></a></li>
+          <li><a href="<%=baseURL%>Tools_and_testing/Client-side_JavaScript_frameworks/Svelte_variables_props"><%=text['Dynamic_behavior_in_Svelte']%></a></li>
```

Comment:
Apologies for taking so long to review this. This PR looks mostly great; the only change I can see that is needed is that the above line has the wrong text reference. It should be `Dynamic_behavior_in_Svelte:_working_with_variables_and_props`.

Once that is fixed, I think we can merge this.